git/o-que-e-controle-de-versao.md

```diff
@@ -0,0 +1,44 @@
+
+Um bom software de controle de versão suporta o workflow preferido do desenvolvedor sem impor uma forma específica de trabalhar. Idealmente, também funciona em qualquer plataforma, em vez de dizer qual sistema operacional ou ferramentas os desenvolvedores devem usar. Ótimos sistemas de controle de versão facilitam um fluxo de mudanças regular e contínuo no código, diferente do que acontece no frustrante e desajeitado mecanismo de bloqueio de arquivos, dando sinal verde a um colaborador, mas ao preço de bloquear o progresso dos outros.
+
+Equipes de software que não usam qualquer forma de controle de versão frequentemente se deparam com problemas como não saber quais mudanças que foram feitas estão disponíveis para os usuários, ou a criação de alterações incompatíveis entre duas partes independentes que precisam então ser meticulosamente desvendadas e retrabalhadas. Se você é um desenvolvedor que nunca usou um controle de versão você pode ter adicionado versões em seus arquivos, talvez com sufixos como "final" ou "mais-recente" e depois teve que lidar com uma nova versão final. Talvez você tenha comentado blocos de código porque você desejava desabilitar certa funcionalidade sem remover o código, temendo que poderia existir uma finalidade para ele mais tarde. O controle de versão é uma saída para esses problemas.
+
+O software de controle de versão tem um papel essencial na rotina dos times de software modernos. Desenvolvedores de software independentes acostumados a trabalhar com um sistema de controle de versão eficiente em seus times, normalmente reconhecem o incrível valor que o controle de versão oferece mesmo em pequenos projetos pessoais. Uma vez acostumados aos eficazes benefícios dos sistemas de controle de versão, muitos desenvolvedores não considerariam trabalhar sem ele, mesmo em projetos não relacionados a software.
+
+## Benefícios dos sistemas de controle de versão
+
+É arriscado desenvolver software sem controle de versão, é o mesmo que não ter backups. O controle de versão pode permitir aos desenvolvedores serem rápidos e aos times de software se manterem eficientes e agéis à medida que a equipe aumenta com a entrada de mais desenvolvedores.
+
+Sistemas de controle de versão (do inglês VCS ou Version Control Systems) tiveram grandes melhorias nas últimas décadas e algumas são melhores do que outras. VCS algumas vezes são conhecidos como ferramentas de gerenciamento de código, ou SCM (do inglês Source Code Management) ou ainda sistema de controle de revisão, ou RCS (do inglês Revision Control System). Uma das mais populares ferramentas de VCS em uso se chama Git. O Git é um VCS <i>Distribuído</i> (ou DVCS), falaremos disso mais tarde. Como muitos do sistemas de VCS disponíveis atualmente, o Git é gratuito e de código aberto. Independentemente do nome que eles recebem, ou qual sistema é usado, os principais benefícios que você deve esperar do controle de versão são os seguintes.
```

Comment:
> `O Git é um VCS <i>Distribuído</i>`

Não precisa usar a tag `<i>` aqui, em markdown, é possível deixar o texto em itálico usando aspas. Ex:

\*Itálico\* => *Itálico*

Reply:
Asterisco né?